src/features/upload-file/providers/base-provider.ts

```diff
@@ -17,19 +17,25 @@ abstract class BaseProvider {
    * this type. We cannot check typeof because there could be a different versions of
    * npm package installed in a project.
    */
-  public name: string
+  public name: string;
 
   /**
    * Bucket is a place where files should be stored. Usually it is a folder location
    */
-  public bucket: string
+  public bucket: string;
+
+  /**
+   * Url is path prefix for local provider
+   */
+  public url: string;
 
   /**
    * @param { string } bucket     place where files should be stored
    */
-  constructor(bucket: string) {
+  constructor(bucket: string, url?: string) {
```

Comment:
It would be better to make second argument an object:
```
, options?: SomeOptionType)
```
then do

```
this.url = options?.url ?? ''
```
If we make it an object, it's later easier to extend it with new options than adding extra argument to constructor

Reply:
done